Sources/MapboxNavigation/RouteLineController.swift

```diff
@@ -153,12 +148,19 @@ extension NavigationMapView {
             }
         }
         
-        func navigationService(_ service: NavigationService, didRefresh routeProgress: RouteProgress) {
+        private func updateRouteLine(routeProgress: RouteProgress, coordinate: CLLocationCoordinate2D?) {
```

Comment:
`updateRouteLine(routeProgress:coordinate:)` is duplicated for iOS and CarPlay, do you think it'll be possible to move it to common place, so that it can be re-used? For example `NavigationMapView`.

Reply:
Please also add basic documentation for this method to describe its purpose.

Reply:
Thanks for the suggestion, I'll add documentation and move this method to `NavigationMapView` to avoid duplication.